ddtrace/profiling/scheduler.py

```diff
@@ -66,6 +67,8 @@ def flush(self):
                 self.before_flush()
             except Exception:
                 LOG.error("Scheduler before_flush hook failed", exc_info=True)
+        # Enable telemetry before the first profile is sent
+        telemetry_writer.app_started()
```

Comment:
IMHO, I recommend moving this to the first line of `flush()`, since that way it'll guarantee to run.  In some configurations (autoinject/SSI !!!), the current line may not execute because some flush configurations can't/don't/won't check for recorders.